docs/installation/requirements.md

```diff
@@ -5,13 +5,13 @@ weight: 1
 
 K3s is very lightweight, but has some minimum requirements as outlined below.
 
-Whether you're configuring a K3s cluster to run in a Docker or Kubernetes setup, each node running K3s should meet the following minimum requirements. You may need more resources to fit your needs.
+Whether you're configuring K3s to run in a container or as a native Linux service, each node running K3s should meet the following minimum requirements. These requirements are baseline for K3s and its packaged components, and do not include resources consumed by the workload itself.
 
 ## Prerequisites
 
 Two nodes cannot have the same hostname.
 
-If all your nodes have the same hostname, use the `--with-node-id` option to append a random suffix for each node, or otherwise devise a unique name to pass with `--node-name` or `$K3S_NODE_NAME` for each node you add to the cluster.
+If multiple nodes may have the same hostname, or if hostnames may be reused by an automated provisioning system, use the `--with-node-id` option to append a random suffix for each node, or devise a unique name to pass with `--node-name` or `$K3S_NODE_NAME` for each node you add to the cluster.
```

Comment:
Grammar nit
```suggestion
If multiple nodes have the same hostname, or if hostnames may be reused by an automated provisioning system, use the `--with-node-id` option to append a random suffix for each node, or devise a unique name to pass with `--node-name` or `$K3S_NODE_NAME` for each node you add to the cluster.
```

Reply:
I wanted the `may` in there because this is the sort of thing you should prepare for... hopefully people plan this out in advance rather than adding the --with-node-id flag after they've already got multiple nodes in the cluster with the same hostname.

How about `will`?

Reply:
I know understand your intent. I think `will` would communicate that better.